### PR TITLE
No featured workshop, no registration, update events

### PR DESCRIPTION
**Pull request by mirisuzanne**
baby sloth

## Related Issue(s)
_Reminder to add related issue(s), if available._

- #634 
- #635

### CODE REVIEW

File: content/workshops/cascading-layouts.md [hunk 1]
```diff
@@ -24,6 +23,11 @@ tags:
   - Container Queries
   - Layout
   - CSSWG
+events:
+  - venue: OddBird Workshop
+    date: 2025-04-07
+    end: 2024-04-09
```
Comment (jgerigmeyer) [inline content/workshops/cascading-layouts.md:29]:
```suggestion
    end: 2025-04-09
```

**Review by jgerigmeyer — APPROVED** [review]

File: content/workshops/cascading-layouts.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
 ---
 title: Cascading Layouts
-sub: A workshop on resilient CSS layouts, June 24-26, 2024
-feature: workshop
+sub: A workshop on resilient CSS layouts
 author: miriam
-date: 2024-06-24
+date: 2024-12-31
```
Comment (jgerigmeyer) [inline content/workshops/cascading-layouts.md:5]: Ensures that this workshop appears first in the list, but without a "2025 Posts" header on Miriam's author page.

File: content/workshops/cascading-styles-deep.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Cascading Style Systems
 sub: A workshop on resilient & maintainable CSS
 author: miriam
-date: 2023-01-09
+date: 2024-10-14
```
Comment (jgerigmeyer) [inline content/workshops/cascading-styles-deep.md:5]: Ensures that this workshop appears earlier in the list.

**Review by jgerigmeyer — COMMENTED** [review]
@mirisuzanne I'm unsure about the changes in https://github.com/oddbird/oddleventy/commit/b5f0fe34313c326f05d031cf586f51d944e2b699, so feel free to revert them if you don't like them.

It means the upcoming workshops will continue to appear first on the Talk and Mia author pages.

**Review by SondraE — CHANGES_REQUESTED** [review]
Need to update the workshop promo blurbs you added to these articles:
/2024/06/13/css-layout/
/2024/06/11/removing-mixins/

**Comment by mirisuzanne** [comment]
@jgerigmeyer those changes look good to me.
@SondraE removed.

**Review by jgerigmeyer — APPROVED** [review]

**Review by SondraE — APPROVED** [review]
lgtm!

**Pull request merged**